cirq-core/cirq/transformers/transformer_primitives_test.py

```diff
@@ -166,6 +166,22 @@ def test_map_operations_respects_tags_to_ignore():
     )
 
 
+def test_apply_tag_to_inverted_op_set():
+    q = cirq.LineQubit.range(2)
+    op = cirq.CNOT(*q)
+    tag = "tag_to_flip"
+    c_orig = cirq.Circuit(op, op.with_tags(tag), cirq.CircuitOperation(cirq.FrozenCircuit(op)))
+    c_flipped_with_deep = cirq.Circuit(
+        op.with_tags(tag), op, cirq.CircuitOperation(cirq.FrozenCircuit(op.with_tags(tag)))
+    )
+    c_flipped_without_deep = cirq.Circuit(
+        op.with_tags(tag), op, cirq.CircuitOperation(cirq.FrozenCircuit(op)).with_tags(tag)
+    )
+    for c_flip, deep in zip([c_flipped_with_deep, c_flipped_without_deep], [True, False]):
```

Comment:
Nit: it feels like this would be simpler to read without the loop.

Reply:
done